lib/python/qmk/keyboard.py

```diff
@@ -72,10 +91,12 @@ def rules_mk(keyboard):
     return rules
 
 
-def render_layout(layout_data, key_labels=None):
+def render_layout(layout_data, render_ascii, key_labels=None):
     """Renders a single layout.
     """
     textpad = [array('u', ' ' * 200) for x in range(50)]
+    style = 'ascii' if render_ascii else 'unicode'
```

Comment:
A thought- it might be more future-proof to do this so we can have other layout styles in the future. Dots? Letters?

```suggestion
def render_layout(layout_data, style='unicode', key_labels=None):
    """Renders a single layout.
    
    Args:
        style
            The style for the box drawing characters. 'ascii' or 'unicode'
            
        key_labels
            A sequence of label strings to write on top of the keys
    """
    textpad = [array('u', ' ' * 200) for x in range(50)]
    style = 'ascii' if render_ascii else 'unicode'
```